requirements.txt

```diff
@@ -1,4 +1,30 @@
 Flask-SQLAlchemy==2.4.1
 SQLAlchemy==1.3.15
+Flask==1.1.1
+awscli==1.18.56
+boto3==1.13.7
```

Comment:
i'm not sure if you need to list all the package dependencies here. if you list the high level ones, the pip command in the docker file will make sure that all the dependencies are installed as well.

Reply:
Good point. I have actually included this for my entire project. This will be required further down the pipeline.